src/replication/log_generator.cpp

```diff
@@ -332,7 +333,10 @@ namespace cubreplication
       }
 
     m_stream_entry.pack ();
+    start_pos = m_stream_entry.get_stream_entry_start_position ();
```

Comment:
I think is better to return stream entry end position.
(when used on context of logging replicated stream position).
Otherwise, when considering the handling of LOG_SYSOP_END_COMMIT_REPLICATED during undo recovery phase, we may consider the entry replicated when is not. (the ack position is performed using end of stream entry).

Reply:
changed to return end position